packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/components/offer-details/translations/Messages_fr_FR.json

```
@@ -0,0 +1,11 @@
+{
+  "dedicatedCloud_offer_upgrade_custom_report": "Rapport de sauvegarde personnalisable",
+  "dedicatedCloud_offer_upgrade_encrypted_backup": "Sauvegarde chiffrée",
+  "dedicatedCloud_offer_upgrade_retention_points": "{{retention}} points de rétention",
+  "dedicatedCloud_offer_upgrade_full_backups": "{{fullBackups}} sauvegardes complètes",
```

What if there is only one ? :)

```suggestion
  "dedicatedCloud_offer_upgrade_full_backup_1": "1 sauvegarde complète",
  "dedicatedCloud_offer_upgrade_full_backup_+": "{{fullBackups}} sauvegardes complètes",
```

---

We could use `<ng-pluralize>` but i dont know if there is `localization` support for it :)

---

```js
<li
                        data-ng-pluralize
                        data-count="CloudProjectDeleteCtrl.resources.instance"
                        data-when="{
                                    'one': ('cloud_project_delete_resource_warning_one_server' | translate),
                                    'other': ('cloud_project_delete_resource_warning_many_server' | translate : {amountInstance: CloudProjectDeleteCtrl.resources.instance})
                                }"
                    ></li>
```

Here is an example from the code

---

Yess i got it. But there is no 1 backup or 1 retention. All plans its above 1...Ill add this anyway

---

Hum if you're sure there is none, you don't have to do it imo 😉